examples/CRISP/apps/server/src/server/utils.rs

```diff
@@ -1,24 +1,78 @@
+use alloy::dyn_abi::{DynSolType, DynSolValue};
```

Comment:
Is there a reason for not using the static encoder?

https://docs.rs/alloy-sol-types/latest/alloy_sol_types/

Reply:
yes, `sol!` it wraps values in a tuple, adding a 32-byte offset at the beginning of the encoding.